closed/src/java.base/share/classes/com/sun/crypto/provider/NativeChaCha20Cipher.java

```diff
@@ -46,6 +46,8 @@
 import sun.security.util.DerValue;
 
 import jdk.crypto.jniprovider.NativeCrypto;
+import jdk.internal.ref.CleanerFactory;
+import java.lang.ref.Cleaner;
```

Comment:
Please organize imports: this import belongs after the import for `VarHandle`.

Reply:
done

Reply:
This doesn't appear to have been addressed; lines 35-36 should read:
```
import java.lang.invoke.VarHandle;
import java.lang.ref.Cleaner;
```
and the import of `CleanerFactory` should not have moved:
```
import jdk.crypto.jniprovider.NativeCrypto;
import jdk.internal.ref.CleanerFactory;
```

Reply:
updated